pkg/nsx/services/mediator/mediator.go

```diff
@@ -32,24 +32,6 @@ type ServiceMediator struct {
 	*node.NodeService
 }
 
```

Comment:
do we still need ServiceMediator struct?

Reply:
it looks like we can remove this mediator file.

Reply:
not now, there is another patch refactoring subnet and subnetset, then we can remove them